### PR TITLE
Disable add collaborators button when limit is reached

### PR DESCRIPTION
**Pull request by midigofrank**
This PR  the `Add collaborators` button if the limit is reached

Depends on: https://github.com/OpenFn/lightning/pull/1984

## Review checklist

- [x] I have performed a **self-review** of my code
- [x] I have verified that all appropriate **authorization policies** (`:owner`, `:admin`, `:editor`, `:viewer`) have been implemented and tested
- [ ] If needed, I have updated the **changelog**
- [ ] Product has **QA'd** this feature

### CODE REVIEW

**Review by jyeshe — COMMENTED** [review]
@stuartc we've double checked yesterday that no database transaction needs to be performed when consuming the event. However like the other events, they might be lost if the GenServer restarts.

The effect would be to allow more collaborators than the would plan accepts.

**Comment by jyeshe** [comment]
@midigofrank could you check the umbrella app tests?

File: test/lightning_web/live/project_live_test.exs [hunk 1]
```diff
@@ -2395,6 +2392,8 @@ defmodule LightningWeb.ProjectLiveTest do
         refute html =~ "You are not authorized to perform this action"
         assert html =~ "Collaborator removed successfully!"
 
+        assert_patch(view, ~p"/projects/#{project}/settings")
```
Comment (jyeshe) [inline test/lightning_web/live/project_live_test.exs:2395]: :pray:

**Review by jyeshe — APPROVED** [review]
:rocket:

**Review by stuartc — APPROVED** [review]

**Pull request merged**